index.js

```diff
@@ -994,29 +1030,20 @@ class GithubScm extends Scm {
      * @async  _getChangedFiles
      * @param  {Object}   config
      * @param  {String}   config.type      Can be 'pr' or 'repo'
-     * @param  {Object}   config.payload   The webhook payload received from the SCM service.
+     * @param  {Object}   [config.payload]   The webhook payload received from the SCM service.
```

Comment:
should change order if this is optional parameter.

Reply:
When `type` is `pr`, this function does'nt need that parameter.